typed_python/Type.hpp

```diff
@@ -1146,15 +1146,6 @@ class Bytes : public Type {
 
 class Value : public Type {
 public:
-    Value(Instance instance) :
```

Comment:
This should be `private` right? (Because we want to always use `Value::Make` to create singleton values)